doc/source/ray-core/patterns/tree-of-actors.rst

```diff
@@ -0,0 +1,28 @@
+Pattern: Using a supervisor actor to manage a tree of actors
+============================================================
+
+In this pattern, a collection of Ray worker actors is managed by a supervisory Ray actor.
```

Comment:
I think we need to start with a description of why you would want to do this. Currently it is not clear what the benefit is.